src/main/data/SamRead.js

```diff
@@ -61,16 +63,39 @@ class SamRead /* implements Alignment */ {
    */
   constructor(buffer: ArrayBuffer, offset: VirtualOffset, ref: string) {
     this.buffer = buffer;
-    this.offset = offset;
+
```

Comment:
@selkovjr: are these changes in SamRead.js an artifact of a problematic merge? If so, would you mind doing a rebase on the master branch? If not, I am not sure if I understand the reasoning behind these changes.